lte/gateway/python/magma/monitord/BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+
+LTE_ROOT = "../../"
+
+ORC8R_ROOT = LTE_ROOT + "../../../orc8r/gateway/python"
```

Comment:
nitpick: Here and elsewhere in this PR.  Using existing DEFINES to build off of them makes a lot of sense if half of a construction is likely to change and is based on another construction (not sure I'm wording that well).  But here, ORCR_ROOT really has no meaningful tie to LTE_ROOT and I would suggest you define ORC8R_ROOT without reference to LTE_ROOT. It implies relationship that doesn't exist?

Reply:
You are right - we should change this here (and in other services when we touch them again - I was about to create general clean-up findings issue anyway).

Reply:
Had a brainstorming in the team because we all are not happy with the current solution. What do you think about the following? - should make it more clear in our opinion

```
MAGMA_ROOT = "../../../../../"
ORC8R_ROOT = "{}orc8r/gateway/python".format(MAGMA_ROOT)
LTE_ROOT = "{}lte/gateway/python".format(MAGMA_ROOT)
```

Reply:
Had a brainstorming in the team because we all are not happy with the current solution. What do you think about the following? - should make it more clear in our opinion

```
MAGMA_ROOT = "../../../../../"
ORC8R_ROOT = "{}orc8r/gateway/python".format(MAGMA_ROOT)
LTE_ROOT = "{}lte/gateway/python".format(MAGMA_ROOT)
```

Reply:
The suggestion would also make "the level" of the Python root compared to the repository root more clear. :+1: 
This might be especially helpful, once/if a bigger code restructuring would be desired / done somewhere in future.

Reply:
I didn't realize such statements were possible in Starlark! But I guess since it's Python based it makes sense that it is capable.  LGTM. Thanks for putting thought into this.